README.md

```diff
@@ -1,3 +1,30 @@
-# MiniBot
+# MiniBot Platform
 
-This repo is a test python implementation of MiniBot.
+This repo is a test python implementation of MiniBot. Original repo can be found
+[here](http://github.com/cornell-cup/cs-minibot-platform). This is a refactored
+version.
+
+## Installing dependencies
+
+Before working with the code, make sure that all dependencies are installed after cloning
+the repository.
+
+Install pylint to use the Python linter.
+
+```
+pip install pylint
+```
+
+To run pylint on the code, run the following line from the root directory.
```

Comment:
Put pylint in like a section about committing or making changes

Reply:
Done (?).